### ISSUE TITLE
ARM board images should be rebuild to unblock the infamous apt stable -> oldstable issue

### ISSUE THREAD

**Issue by mikerjensen** [YunoHost/issues#1871]
Hi, FYI, I just installed the latest RPI Yunohost image downloaded yesterday, and then tried to install a package but it said it can't install because a later version of Yunohost is needed. So then I went to update the system packages through the Ynh interface and got:

```
Unable to update the cache of APT (Debian's package manager). Here is a dump of the sources.list lines, which might help identify problematic lines:
sources.list:deb http://raspbian.raspberrypi.org/raspbian/ buster main contrib non-free rpi
sources.list.d/yunohost.list:deb http://forge.yunohost.org/debian/ buster stable
sources.list.d/raspi.list:deb http://archive.raspberrypi.org/debian/ buster main

E: Repository 'http://raspbian.raspberrypi.org/raspbian buster InRelease' changed its 'Suite' value from 'stable' to 'oldstable'
```

So i ssh'd into the RPI3 and did an 'apt update' - as per the below:
```
#apt update
Hit:1 http://archive.raspberrypi.org/debian buster InRelease                   
Hit:2 http://forge.yunohost.org/debian buster InRelease                        
Get:3 http://raspbian.raspberrypi.org/raspbian buster InRelease [15.0 kB]
E: Repository 'http://raspbian.raspberrypi.org/raspbian buster InRelease' changed its 'Suite' value from 'stable' to 'oldstable'
N: This must be accepted explicitly before updates for this repository can be applied. See apt-secure(8) manpage for details.
Do you want to accept these changes and continue updating from this repository? [y/N] y
```

Updating was fine via this method. BTW. the stock RPI2/3/4 image for Yunohost doesn't seem to work on my RPI4, only the RPI3...

**Comment by tituspijean** [comment]
For reference: https://forum.yunohost.org/t/updates-may-fail-because-repository-changed-its-suite-value-from-stable-to-oldstable/16931

I guess we can close this issue and label it as "wont fix". :/

**Comment by alexAubin** [comment]
(The fix is that we should rebuild fresh images, which is also relevant for internet cubes because this miserably crashes the install process)

**Comment by alexAubin** [comment]
Closing because duplicate of https://github.com/YunoHost/issues/issues/1852
